tracing.md

```diff
@@ -1,17 +1,44 @@
-# OpenTelemetry Tracing with Jaeger
+# OpenTelemetry Tracing
 Gubernator supports [OpenTelemetry](https://opentelemetry.io) for generating
 detailed traces of application behavior and sending to [Jaeger
 Tracing](https://www.jaegertracing.io/) server.
 
-Read more at:
-[https://github.com/mailgun/holster/blob/master/tracing/README.md](https://github.com/mailgun/holster/blob/master/tracing/README.md)
-
 ## Enabling Jaeger Exporter
```

Comment:
I would just redirect to `holster`'s `tracing/README.md`.

Reply:
I want to have something here, in the off chance something happens to holster or one of it's many libraries.